sql/step_82_drop_deleted.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE oh_admission DROP COLUMN adm_deleted;
+ALTER TABLE oh_patient DROP COLUMN pat_deleted;
+ALTER TABLE oh_supplier DROP COLUMN sup_deleted;
```

Comment:
Sorry, just realized that:

The script, before to deleted those fields, should also "migrate" previous data, by setting `XXX_ACTIVE=0` for all `XXX_DELETED='Y'`.

`load_demo_data.sql` should also be provided.